editor/src/templates/project-not-found/ProjectNotFound.tsx

```diff
@@ -1,15 +1,21 @@
 /** @jsxRuntime classic */
 /** @jsx jsx */
-import React from 'react'
 import { jsx } from '@emotion/react'
-import { Button } from '../../uuiui/button'
+import React from 'react'
 import { UTOPIA_BACKEND_BASE_URL } from '../../common/env-vars'
-import { when } from '../../utils/react-conditionals'
 import { requestProjectAccess } from '../../components/editor/server'
+import { when } from '../../utils/react-conditionals'
+import { Button } from '../../uuiui/button'
 
 const PyramidLight404 = `${process.env.UTOPIA_DOMAIN}/editor/404_pyramid_light.png?hash=${process.env.UTOPIA_SHA}`
 
-export default function ProjectNotFound({ projectId }: { projectId?: string }) {
+export default function ProjectNotFound({
+  projectId,
+  loggedIn,
+}: {
+  projectId?: string
+  loggedIn?: boolean
```

Comment:
Do these need to be optional?

Reply:
nah I don't think they do – I adapted to the existing `?:` but it makes sense to have them explicit

Reply:
fixed in [`be43df5` (#5054)](https://github.com/concrete-utopia/utopia/pull/5054/commits/be43df5e02c673827cff314efb0900d231fbca39)